### PR TITLE
[2018-10] [monodroid] Fill-in reference stubs for System.Drawing types that are …

### PR DESCRIPTION
**Pull request by alexischr**
…currently 'forked' from Mono into Mono.Android.dll

The list of localy implemented System.Drawing types: https://github.com/xamarin/xamarin-android/blob/e0da1f152de84c4cc39983f2fda775ec98c08250/src/Mono.Android/Mono.Android.csproj#L311-L325

### CODE REVIEW

File: mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/PointF.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Rectangle.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/RectangleF.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/Size.cs
+../../../../external/corefx/src/System.Drawing.Primitives/src/System/Drawing/SizeF.cs
+../../../../external/corefx/src/Common/src/System/Numerics/Hashing/HashHelpers.cs
+../../../../external/corefx/src/Common/src/System/Drawing/KnownColorTable.cs
+../../../../external/corefx/src/Common/src/System/Drawing/ColorTable.cs
+../../../../external/corefx/src/Common/src/System/Drawing/ColorUtil.netcoreapp21.cs
+../../../build/common/SR.cs
```
Comment (alexischr) [inline mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources]: There must be a better way to include `mcs/build/common/SR.cs`, maybe it's auto-included if this weren't a facade?

**Comment by marek-safar** [comment]
You need to add TypeForwarderFrom

File: mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources (hunk 1 above)
Comment (marek-safar) [reply mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources]: that's what we do but not sure we actually need it here

Comment (alexischr) [reply mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources]: `SR.Format` is used by `Color.cs`

Comment (marek-safar) [reply mcs/class/Facades/System.Drawing.Common/monodroid_System.Drawing.Common.dll.sources]: ok

**Comment by alexischr** [comment]
@marek-safar you mean from `Mono.Android.dll`?

**Comment by marek-safar** [comment]
no, it needs to be here

**Comment by marek-safar** [comment]
@monojenkins backport 2019-02

**Comment by marek-safar** [comment]
@monojenkins backport master

**Comment by marek-safar** [comment]
@alexischr please backport manually

**Comment by alexischr** [comment]
@monojenkins build failed

**Comment by alexischr** [comment]
@monojenkins build Linux x64

**Comment by alexischr** [comment]
@monojenkins build Linux x64

**Comment by alexischr** [comment]
@monojenkins build OS X x64 Android SDK

**Comment by alexischr** [comment]
@monojenkins commit csproj and apidiff

**Pull request merged**